metadata/stream/sample.json

```diff
@@ -56,7 +56,8 @@
               "disk": {
                 "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190905.0/x86_64/fedora-coreos-30.20190905.0-openstack.qcow2.xz",
                 "signature": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190905.0/x86_64/fedora-coreos-30.20190905.0-openstack.qcow2.xz.sig",
-                "sha256": "7b6608f03bcf98f41494c0a71fa518256798065c2516ff757e6bdd766f870ede"
+                "sha256": "7b6608f03bcf98f41494c0a71fa518256798065c2516ff757e6bdd766f870ede",
+                "uncompressed-sha256": "38acb15d02d5ac0f2a2789602e9df950c380d2799b4bdb59394e4eeabdd3a662"
```

Comment:
Here too, but meh we can just refresh that file from real output once we have it to make it easier.

Reply:
Should be able to copy-paste from https://builds.coreos.fedoraproject.org/streams/stable.json now!